nestargs/decorators.py

```diff
@@ -6,7 +6,11 @@
 def option(parameter_name, **arg_params):
     def decorator(f):
         sig = inspect.signature(f)
-        if parameter_name not in sig.parameters:
+        if (
+            parameter_name not in sig.parameters
+            and inspect.Parameter.VAR_KEYWORD
+            not in (sig.parameters[k].kind for k in sig.parameters.keys())
```

Comment:
In this case, the set comprehension is semantically more correct than the generator comprehension.

```suggestion
            not in {sig.parameters[k].kind for k in sig.parameters.keys()}
```